test/objective-c/EnvoyBridgeUtilityTest.m

```diff
@@ -0,0 +1,29 @@
+#import <XCTest/XCTest.h>
+
+typedef NSDictionary<NSString *, NSArray<NSString *> *> EnvoyHeaders;
+
+typedef NSDictionary<NSString *, NSString *> EnvoyTags;
```

Comment:
Are these necessary?

Reply:
They are, unfortunately; this is a side effect of the bridge header issue.